packages/scripts/src/helpers/test-runner/services.ts

```diff
@@ -20,6 +20,11 @@ const logger = ts.debugLogger('ts-scripts:cmd:test');
 
 const serviceUpTimeout = ms('2m');
 
+const rabbitConfigPath = `${getRootDir()}/.ts-test-config/rabbitmq.conf`;
```

Comment:
We should use `path.join` here:

```suggestion
const rabbitConfigPath = path.join(getRootDir(), '.ts-test-config/rabbitmq.conf');
```